public/app/plugins/datasource/prometheus/querybuilder/components/PromQueryEditorSelector.tsx

```diff
@@ -101,6 +106,7 @@ export const PromQueryEditorSelector = React.memo<Props>((props) => {
         {editorMode === QueryEditorMode.Builder && (
           <>
             <QueryHeaderSwitch label="Raw query" value={rawQuery} onChange={onQueryPreviewChange} />
+            <QueryHeaderSwitch label="Explain" value={showExplain} onChange={onShowExplainChange} />
```

Comment:
You are adding Explain to Code editor, but because these switches are only in builder, you can not interact with it in code editor. In loki, I've moved Explain switch out of this section and added it before Raw query.

Reply:
I've addressed this in here https://github.com/grafana/grafana/pull/52935/commits/b62d8cc1d66b95055d88701db44dfb62152e861b

Reply:
I see what you've meant. I realized it while writing tests. So we want the explain mode in code mode too. I will update again.